elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -149,9 +149,10 @@ def _update_advisory(runtime, advisory, flaw_bugs, bug_tracker, noop):
     bug_tracker.attach_bugs(advisory_id, flaw_ids, noop)
 
 
-async def associate_builds_with_cves(errata_api: AsyncErrataAPI, advisory: Erratum, flaw_bugs: Iterable[Bug], attached_tracker_bugs: List[Bug], tracker_flaws: Dict[int, Iterable], dry_run: bool):
+async def associate_builds_with_cves(errata_api: AsyncErrataAPI, advisory: Erratum, flaw_bugs: Iterable[Bug],
+                                     attached_tracker_bugs: List[Bug], tracker_flaws: Dict[int, Iterable], dry_run: bool):
     attached_builds = [b for pv in advisory.errata_builds.values() for b in pv]
-    cve_components_mapping: Dict[str, Set[str]] = {}
+    cve_components_mapping: Dict[str, Dict] = {}
```

Comment:
We now need the flaw bug object to get the information to associate the right builds with each flaw bug.